### PR TITLE
Xlsx Reader

### PR DESCRIPTION
**Pull request by 1owkeyme**
# Changelog

### Added

- Xlsx Reader

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+        <dependency>
+            <groupId>org.apache.poi</groupId>
+            <artifactId>poi</artifactId>
+            <version>5.3.0</version>
+        </dependency>
+
+
+    </dependencies>
+</project>
```
[the remark this answers is not in the document]
Comment (1owkeyme) [reply pom.xml:81]: Я использую дефолтный xml форматтер. В репе maven, например, нет пустой строки  https://github.com/apache/maven/blob/master/pom.xml

File: src/main/java/com/xlsx_reader/service/XlsxReaderService.java [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+        if (iban == null || bic == null || accountHolder == null) {
+            throw new ExcelParsingException("Incomplete bank account data on row: " + row.getRowNum());
+        }
+        return new BankAccount(iban, bic, accountHolder);
+    }
+
+    private String getCellValue(Row row, int columnIndex) {
+        Cell cell = row.getCell(columnIndex);
+        return (cell != null) ? cell.toString() : null;
```
[the remark this answers is not in the document]
Comment (1owkeyme) [reply src/main/java/com/xlsx_reader/service/XlsxReaderService.java]: 11c4d4ab5fee947cdca3e6ec173cf6b18efc34cd

File: src/main/java/com/xlsx_reader/exception/InvalidFilePathException.java [hunk 3]
```diff
@@ -0,0 +1,7 @@
+package com.xlsx_reader.exception;
+
+public class InvalidFilePathException extends Exception {
+    public InvalidFilePathException(String message) {
+        super(message);
+    }
+}
```
[the remark this answers is not in the document]
Comment (1owkeyme) [reply src/main/java/com/xlsx_reader/exception/InvalidFilePathException.java]: f3d5d3f1d1cbedb1be435e2a19a3c9233f869c6d

**Comment by 1owkeyme** [comment]
@tanabahh,all threads have been addressed, feel free to review them at your convenience, or simply let me know if I do the merge without reviewing

File: src/main/java/com/xlsx_reader/service/XlsxReaderService.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (1owkeyme) [reply src/main/java/com/xlsx_reader/service/XlsxReaderService.java]: d820d73d56d2b91089cbd3352e4f37c54340d23d

[the remark this answers is not in the document]
Comment (1owkeyme) [reply src/main/java/com/xlsx_reader/service/XlsxReaderService.java]: d1169af9b7e5901dffdd21c3d23aed9492ee2fac - tests fix

**Comment by tanabahh** [comment]
Everything is great:) Thank you for your work! You can merge it

**Pull request merged**